public/app/features/alerting/unified/api/alertRuleApi.ts

```diff
@@ -433,7 +433,7 @@ export const alertRuleApi = alertingApi.injectEndpoints({
       },
       providesTags: ['DeletedRules'],
     }),
-    removePermanentlyDeletedRule: build.mutation<void, { guid: string }>({
+    permanentlyDeleteRule: build.mutation<void, { guid: string }>({
```

Comment:
I don't think we were removing a permanently deleted rule, if anything it was removing a recently deleted rule?